frontend/src/components/authorization/AuthorizationBasicInfoProponent.vue

```diff
@@ -0,0 +1,92 @@
+<script setup lang="ts">
+import { ref, watchEffect } from 'vue';
+import { useI18n } from 'vue-i18n';
+
+import { userService } from '@/services';
+import { formatDate } from '@/utils/formatters';
+
+import type { Ref } from 'vue';
+import type { Permit } from '@/types';
+
+// Props
+const { permit } = defineProps<{
+  permit: Permit;
+}>();
+
+// Composables
+const { t } = useI18n();
+
+// State
+const cardUpdatedBy: Ref<string> = ref('');
+
+// Actions
+watchEffect(() => {
+  if (permit.updatedBy) {
+    userService
+      .searchUsers({ userId: [permit.updatedBy] })
+      .then((res) => {
+        cardUpdatedBy.value = res?.data.length ? res?.data[0].fullName : '';
```

Comment:
Is this being used? Also proponents aren't able to make this service call with their permissions.